static/output.js

```diff
@@ -30,7 +30,6 @@ define(function (require) {
     var FontScale = require('fontscale');
 
     function Output(hub, container, state) {
-        var self = this;
```

Comment:
My editor was warning about unused variables. So thats what the two below are :)